db/models/score/scores.py

```diff
@@ -0,0 +1,40 @@
+"""The module containing all code related to the `scores` table
+within the Postgres db.
+"""
+import uuid
+from db import db
+from sqlalchemy import ForeignKey 
+from sqlalchemy.dialects.postgresql import UUID
+
+
+class JustScore(db.Model):
+    """Score The sqlalchemy-flask model class used to define the
+    `scores` table in the Postgres database."""
+
+    __tablename__ = "just_scores"
+
+    just_score_id = db.Column(
+        "score_id", UUID(as_uuid=True), default=uuid.uuid4, primary_key=True
+    )
+
+    score = db.Column(
+        "score", db.Integer(), nullable=False
+    )
+
+    justification = db.Column(
+        "justification", db.Text(), nullable=False
+    )
+
+    application_id = db.Column(
+        "application_id", UUID, ForeignKey("assessment_records.application_id")
+    )
+
+    timestamp = db.Column(
```

Comment:
Can we make this `date_created` rather than `timestamp`?

Reply:
have changed this now 👍🏾